Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
@KuoE0 better not to change the formula installation process, just updating the version seems fine. People using `pastebin` service with this client, might be using their own unique `api_dev_key` and would not want that to be overwritten.